src/SDKs/Consumption/Management.Consumption/Microsoft.Azure.Management.Consumption.csproj

```diff
@@ -6,7 +6,7 @@
     <PropertyGroup>
         <PackageId>Microsoft.Azure.Management.Consumption</PackageId>
         <Description>Microsoft Azure Consumption Management Library</Description>
-        <VersionPrefix>2.1.0</VersionPrefix>    
+        <VersionPrefix>3.0.0</VersionPrefix>    
```

Comment:
you are missing package release notes.
This is important because the user will know what is in the new release.
Look at other csproj and you will know how to provide < packageReleaseNotes >